stdlib/bigint.c

```diff
@@ -601,4 +601,4 @@ int sdivmod512(uint512_t *pdividend, uint512_t *pdivisor, uint512_t *remainder,
     }
 
     return 0;
-}
+}
```

Comment:
Please don't modify files that are not relevant for the PR (that includes the `stdlib/wasm/bigint.bc` and `stdlib/bpf/bigint.bc`.